src/VisualStudio/Core/Def/Utilities/ClipboardHelpers.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+
+namespace Microsoft.VisualStudio.LanguageServices.Utilities
+{
+    internal static class ClipboardHelpers
+    {
+        [DllImport("ole32.dll", ExactSpelling = true, CharSet = CharSet.Unicode)]
+        private static extern int OleGetClipboard(out IDataObject dataObject);
```

Comment:
This will have to be the native IDataObject, not the one from WPF or WinForms.

Reply:
Changed to be the correct IDataObject